sources/models/activities.js

```diff
@@ -1,10 +1,23 @@
+const dateToStr = webix.Date.dateToStr("%Y-%m-%d %H:%i");
+
 export const activities = new webix.DataCollection({
 	url: "http://localhost:8096/api/v1/activities/",
 	save: "rest->http://localhost:8096/api/v1/activities/",
 	scheme: {
 		$init(item) {
-			item.DueDate = webix.i18n.parseFormatDate(item.DueDate);
-			item.DueTime = webix.Date.timePart(item.DueDate);
+			if (item.DueDate) {
+				item.DueDate = webix.i18n.parseFormatDate(item.DueDate);
+			}
+		},
+		$update: item => {
+			const dateToStrDate = webix.Date.dateToStr("%Y-%m-%d");
+			const dateToStrTime = webix.Date.dateToStr("%H:%i");
+			item.DueDate = `${dateToStrDate(item.DueDate)} ${dateToStrTime(item.DueTime)}`;
```

Comment:
At this moment the Date object becomes a string in client-side data ([log](http://prntscr.com/ri2bhk) before and after editing). This will affect the filtering and sorting (they would expect a Date).

To maintain `DueDate` as a Date object, use the related API to set the needed time. Something like that:
```
let hours = DueTime.getHours(),
    minutes = DueTime.getMinutes();
DueDate.setHours(hours, minutes)
```

Reply:
Fixed.